CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `skip` and `trim_duration` options
 - `export_mvs` option
 - `pkt_skip_mod` option
+- `nmd_set_drop_ref()` (was disabled)
```

Comment:
`was disabled` -> `unimplemented` (and same goes to the commit msg) ?

Reply:
sure, done